changelog/fragments/operator-name-config-file.yaml

```diff
@@ -0,0 +1,10 @@
+entries:
+  - description: >
+      Added `project-name` key to the PROJECT config file (v3-alpha+).
+    kind: addition
+    breaking: false
+    migration:
+      header: Add the `project-name` config key to your PROJECT file
+      body: >
+        Set the `project-name` key in your PROJECT file. If this key is not set,
+        the current working directory's base name will be used in various subcommands.
```

Comment:
we do not need a fragment in this case. it is part of the new helm layout.

Reply:
And Go's, which was in v0.19.

Reply:
It is true 👍